### PR TITLE
Describe emoji with emoji

### PR DESCRIPTION
**Pull request by monich**
And stickers too. In other words, replace this:

Screenshot_20211207_002

with this:

Screenshot_20211207_001

What do you think?

### CODE REVIEW

File: qml/js/functions.js [hunk 1]
```diff
@@ -47,9 +47,9 @@ function getMessageText(message, simple, currentUserId, ignoreEntities) {
             return enhanceMessageText(message.content.text, ignoreEntities);
         }
     case 'messageSticker':
-        return simple ? qsTr("Sticker: %1").arg(message.content.sticker.emoji) : "";
+        return message.content.sticker.emoji
     case 'messageAnimatedEmoji':
-        return simple ? qsTr("Animated Emoji: %1").arg(message.content.animated_emoji.sticker.emoji) : "";
+        return message.content.animated_emoji.sticker.emoji
```
Comment (Wunderfitz) [inline qml/js/functions.js:52]: This change causes sticker emojis and animated emojis to duplicate in chats. You should still check for the `simple` parameter.

**Review by Wunderfitz — CHANGES_REQUESTED** [review]
Also thought about that after the 0.11 release. ;) So in general, I support the change, but in functions.js, you should still check for the `simple` parameter.

File: qml/js/functions.js (hunk 1 above)
Comment (monich) [reply qml/js/functions.js:52]: Right! Good catch.

Comment (monich) [reply qml/js/functions.js]: Fixed!

**Comment by monich** [comment]
And I'll probably add an option to show animated emoji as a sticker rather than an image. But that would be a separate patch.

**Review by Wunderfitz — APPROVED** [review]
Thanks for that! :)

**Pull request merged**